src/cli/log.rs

```diff
@@ -20,7 +20,7 @@ use crate::{process::Process, utils::notify::NotificationLevel};
 pub fn tracing_subscriber(
     process: &Process,
 ) -> (
-    impl tracing::Subscriber,
+    impl tracing::Subscriber + use<'_>,
```

Comment:
Are the `use` blocks necessary for compilation to succeed?

Reply:
Yes, `impl Foo + use<A>` is there to indicate that `impl Foo` is only involved with `A` and nothing else, otherwise the 2024 Language Edition will deduce that you'll also need all unused generic parameters such as `'self`. In our case this will cause overly restrictive bounds.

Reply:
Wait, I've probably done it wrong: the bound on `telemetry()` is the one that should be fixed, not the one on the final logger. I'll have another look.

Reply:
Yes, `clippy` is doing it right this time :)